tests/github/export-diff.js

```diff
@@ -21,113 +27,135 @@ pullRequest.checkEnv()
+        );
+
+        // remove duplicates
+        return task === firstEqualTask;
+      })
+      .sort((a, b) => {
+        const manFixCompare = a.manFix.localeCompare(b.manFix);
+        const pluginCompare = a.pluginKey.localeCompare(b.pluginKey);
+
+        if (manFixCompare !== 0) {
+          return manFixCompare;
+        }
+
+        return pluginCompare;
+      });
+
+    return tasks;
+
+    /**
+     * @returns {!Array.<Task>} What export diff tasks have to be done due to changes in the model. May be empty.
+     */
+    function getTasksForModel() {
+      let tasks = [];
+
+      if (changedComponents.added.model ||
+        changedComponents.modified.model ||
+        changedComponents.renamed.model ||
+        changedComponents.removed.model) {
+
+        for (const manFix of usableTestFixtures) {
+          tasks = tasks.concat(usablePlugins.map(pluginKey => ({
+            manFix,
+            pluginKey
+          })));
+        }
+      }
 
-    if (changedComponents.modified.model || changedComponents.renamed.model) {
-      lines = lines.concat(getModelMessage(allPlugins, allTestFixtures));
+      return tasks;
     }
-    else {
-      for (const plugin of changedComponents.modified.exports) {
-        lines = lines.concat(getPluginMessage(plugin, allTestFixtures));
+
+    /**
+     * @returns {!Array.<Task>} What export diff tasks have to be done due to changes in plugins. May be empty.
+     */
+    function getTasksForPlugins() {
+      let tasks = [];
+
+      const changedPlugins = changedComponents.added.exports.concat(changedComponents.modified.exports);
+
+      for (const changedPlugin of changedPlugins) {
+        tasks = tasks.concat(usableTestFixtures.map(manFix => ({
+          manFix,
+          pluginKey: changedPlugin
+        })));
       }
-    }
 
-    for (const fixture of changedComponents.modified.fixtures) {
-      lines = lines.concat(getFixtureMessage(allPlugins, `${fixture[0]}/${fixture[1]}`));
+      return tasks;
     }
 
-    if (lines.length > 0) {
-      lines = [
-        `You can run view your uncommited changes in plugin exports manually by executing:`,
-        `\`$ node cli/diff-plugin-outputs.js -p <plugin name> <fixtures>\``,
-        ``
-      ].concat(lines);
+    /**
+     * @returns {!Array.<Task>} What export diff tasks have to be done due to changes in fixtures. May be empty.
+     */
+    function getTasksForFixtures() {
+      let tasks = [];
+
+      const fixtures = changedComponents.added.fixtures.concat(changedComponents.modified.fixtures, changedComponents.renamed.fixtures);
+
+      for (const [manKey, fixKey] of fixtures) {
+        tasks = tasks.concat(usablePlugins.map(pluginKey => ({
+          manFix: `${manKey}/${fixKey}`,
+          pluginKey
+        })));
+      }
+
+      return tasks;
     }
+  })
+  .then(tasks => {
+    let resultLines = [];
+    tasks.forEach(task => {
+      resultLines = resultLines.concat(performTask(task));
+    });
 
```

Comment:
If `resultLines` is too long to fit in a single GitHub comment, add a warning like 

#### :warning: The output of the script is too long to fit in this comment, please run it yourself locally!

and truncate the comment (this could be difficult if you don't want to have dangling open HTML tags, though).